context/app/static/js/shared-styles/buttons/OutlinedButton/OutlinedButton.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { StyledButton } from 'js/shared-styles/buttons/OutlinedButton/style';
+import { ButtonProps } from '@mui/material/Button';
+
+interface OutlinedButtonProps extends ButtonProps {
+  children: React.ReactNode;
+}
+
+function OutlinedButton({ children, ...rest }: OutlinedButtonProps) {
+  return (
+    <StyledButton variant="outlined" {...rest}>
```

Comment:
Not sure we need to introduce a new shared component to pass a single prop. @NickAkhmetov thoughts?

Reply:
Good point - this was created because I believed the `variant` styling could only be applied inline, and the outline is always necessary for this component. I just wrapped the `Button` component into the `styled` function so that the `variant` styling can be applied there, which eliminates the need for this component.